composites/Plugin/Shared/components/Button.js

```diff
@@ -268,15 +268,23 @@ IconButton.defaultProps = {
  * @returns {ReactElement} Styled icon button.
  */
 export const IconsButton = ( props ) => {
-	const { children: text, prefixIcon, prefixIconColor, suffixIcon, suffixIconColor } = props;
-
-	const newProps = omit( props, [ "prefixIcon", "prefixIconColor", "suffixIcon", "suffixIconColor" ] );
+	const {
+		children: text,
```

Comment:
You rename the children prop explicitely to text, but the `propTypes` of the component suggest it can also be a element or multiple elements.

Reply:
No I haven't touched this part, it was already `children: text`